SPECS/kubevirt/Cleanup-housekeeping-cgroup-on-vm-del.patch

```diff
@@ -0,0 +1,249 @@
+From 16c98473d5b3e80b65eda99b0d135669c2d7635e Mon Sep 17 00:00:00 2001
+From: Kanika Nema <kanikanema@microsoft.com>
+Date: Fri, 5 May 2023 18:19:59 +0000
+Subject: [PATCH] Cleanup housekeeping cgroup on vm delete
```

Comment:
I think we should propose this patch to the upstream if its not specific to us.

Reply:
Yes, that is the plan but it will take sometime. We want this in the May release of Mariner hence adding it as a patch.